docs/reference/api/metadata-stores/delta.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Delta Lake Metadata Store
+---
+
+# DeltaMetadataStore
+
+Delta Lake is the recommended object-store-backed metadata backend.
+It keeps one Delta table per feature, appends new metadata versions, and leans on Polars / Narwhals for provenance calculations.
+
+## Installation
+
+The backend relies on [`deltalake`](https://delta-io.github.io/delta-rs/python/)
+which ships with Metaxy’s `delta` dependency group:
+
+```bash
+uv sync --group delta
+```
+
+## Object Stores
+
+Point `root_path` at any supported URI (`s3://`, `abfss://`, `gs://`, `file://`,
+`local://`, …) and forward credentials with `storage_options`.
+The dict is passed
+verbatim to `deltalake`, so you can reuse the same keys you would use with delta-rs directly.
+
+## Storage Layout
+
+Control how feature keys map to Delta directories with `layout`:
+
+- `flat` (default) keeps the current `<namespace>__<feature>` directory scheme.
+- `nested` places every key part into its own directory (e.g.
+  `namespace/feature/channel`), which can make large hierarchies easier to
+  inspect in object-store consoles.
+
+Only URI generation changes; table contents remain identical.
+
+## Hashing
```

Comment:
There is no need to repeat it here, it's already obvious from API docs

Reply:
removed